### PR TITLE
Added a new method to retrieve the state of a single item and updated…

### PR DESCRIPTION
**Pull request by legovaer**
In this PR I added a new method that allows you to retrieve the information from a single item using:

`niko.item_state(<id>)`

I also updated the README file accordingly.

### CODE REVIEW

File: nikohomecontrol/__init__.py [hunk 1]
```diff
@@ -34,6 +34,13 @@ def list_locations(self):
     def execute_actions(self, id, value):
         return self._command('{"cmd":"executeactions", "id": "'+id+'", "value": "'+value+'"}')
 
+    def item_state(self, id):
+        data = self.list_actions()['data']
+        for index, item in enumerate(data):
+            if 'id' in item and str(item['id']) == str(id):
+                return item
+        return False
```
Comment (NoUseFreak) [inline nikohomecontrol/__init__.py:42]: Can you raise an error if the id is not found. So this function returns the same type.

**Comment by NoUseFreak** [comment]
@legovaer Did you by any chance validate this against the v2 of the home control version. I'm currently still running the v1 version at my house. Just wondering if I can update my niko without having to rewrite this :-)

**Comment by legovaer** [comment]
@NoUseFreak I'm running v2 at my house and works like a charm. I'm also working on the HASS component and achieved some decent results. I'm now doing some refactoring to allow HASS to listen to NHC events as well :)

**Comment by NoUseFreak** [comment]
@legovaer I have a hass component that sort of works, it's a little laggy, maybe you can have a look if you want. 
https://github.com/NoUseFreak/home-assistant.github.io/tree/feature/niko-home-control
https://github.com/NoUseFreak/home-assistant/tree/feature/niko-home-control

**Comment by NoUseFreak** [comment]
Closing because PR has gone stale

**Pull request closed without merging**